python_modules/dagster/dagster/core/object_store.py

```diff
@@ -89,13 +89,39 @@ def set_value(self, obj, context, runtime_type, paths):
             return self.TYPE_REGISTRY[runtime_type.name].set_object(
```

Comment:
I think we should rename this to TYPE_STORAGE_PLUGIN_REGISTRY or something for clarity

Reply:
👍